modpack_format/implementing.md

```diff
@@ -0,0 +1,31 @@
+# Implementing the Modpack Format
+This is a guide for launchers on how to implement the modpack format.
```

Comment:
Again with the semantics, but: how to *use* the format

Reply:
Resovled